go.opentelemetry.io/vanity.yaml

```diff
@@ -1,7 +1,11 @@
 host: go.opentelemetry.io
 
 paths:
-  /otel:
+  /lib:
     repo: https://github.com/open-telemetry/opentelemetry-go
   /collector:
     repo: https://github.com/open-telemetry/opentelemetry-collector
+  /contrib/lib:
+    repo: https://github.com/open-telemetry/opentelemetry-go-contrib
+  /contrib/collector:
```

Comment:
Would rather `/collector/contrib`

Reply:
Then otel/contrib for the library.

Reply:
Also it is not possible I think to use collector/contrib because that causes a problem if collector repo defines a contrib package.

Reply:
> Also it is not possible I think to use collector/contrib because that causes a problem if collector repo defines a contrib package.
good point.

Does `/collector-contrib` work?

Reply:
Ah, nvm. Just re-read. I think the proposed changes make sense.

Reply:
Let's avoid having a `contrib` directory in the collector, then?

Reply:
Why? Is the name really a problem? Do we need to deal with this possible problem?